concrete/src/File/Image/Thumbnail/Type/Version.php

```diff
@@ -14,14 +14,16 @@ class Version
     protected $directoryName;
     protected $handle;
     protected $name;
+    protected $sizingMode;
     protected $width;
     protected $height;
     protected $isDoubledVersion;
 
-    public function __construct($directoryName, $handle, $name, $width, $height, $isDoubledVersion = false)
+    public function __construct($directoryName, $handle, $name, $sizingMode, $width, $height, $isDoubledVersion = false)
```

Comment:
This still breaks BC because if anyone has extended this, their code will break, if anything this parameter should be added as an optional parameter at the end of the function signature as PHP will ignore extra parameters